### PR TITLE
Post startup

### PR DESCRIPTION
**Pull request by catfact**
this change moves the execution of the mods post-startup hook until after initially-connected external devices have been scanned by `dev_monitor` and handled by appropriate lua modules.

its sort of a quick hack so please take a look and see if anything looks crazy!

also could easily be convinced that this is not necessary since there are plenty of workarounds available. (e.g. mod could monkeypatch `_norns.midi.add`)

### CODE REVIEW

**Comment by catfact** [comment]
(wow github is using my fake email field to determine the contributor for these commits. how stupid)

File: lua/core/norns.lua [hunk 1]
```diff
@@ -237,3 +237,8 @@ end
 _startup = function()
   require('core/startup')
 end
+
+_post_startup = function()
+   print('_norns._post_startup')
+   hook.system_post_startup()
```
Comment (ngwese) [inline lua/core/norns.lua:243]: i get slightly weirded out by references to a module in a file which doesn’t import it. i realize that pattern is all over the core code. iirc the `mods` does import `hooks` but i double it is importing it globally and calling it `hook`.

File: matron/src/main.c [hunk 2]
```diff
@@ -83,23 +83,44 @@ int main(int argc, char **argv) {
...
     // i/o subsystems are ready; run user startup routine
     w_startup();
+
     // scan for connected input devices
+    fprintf(stderr, "scanning devices...\n");
     dev_monitor_scan();
 
+    // handle all resulting events, then run "post-startup"
+    fprintf(stderr, "handling pending events...\n");
+    //event_handle_pending();
```
Comment (ngwese) [inline matron/src/main.c:118]: is event_handle_pending intended to be commented out?

**Review by ngwese — COMMENTED** [review]
took a quick look (on my phone so couldn’t test)

File: matron/src/main.c (hunk 2 above)
Comment (catfact) [reply matron/src/main.c:118]: oops no

File: lua/core/norns.lua (hunk 1 above)
Comment (catfact) [reply lua/core/norns.lua:243]: yea i agree. but we constantly assume the global names assigned in `startup`. hppy to do my part to change that style

Comment (catfact) [reply lua/core/norns.lua:243]: ACTUally nvm i am locally importing `hook` b/c i am a good boy

Comment (catfact) [reply lua/core/norns.lua:243]: actually nvm hook is a local alias, i'm a good boy

Comment (ngwese) [reply lua/core/norns.lua:243]: sure enough. sorry for the false alarm (hard to review on the phone)

**Review by ngwese — APPROVED** [review]
gave a quick test on device. lgtm

**Pull request merged**